puns.json

```diff
@@ -310,5 +310,25 @@
   {
     "pun": "Q: What does a programmer say after a road trip across California?",
     "punchline": "A: It was a solid state drive"
+  },
+  {
+    "pun": "Q: What do Javascript programmers like to order at the coffeeshop?",
+    "punchline": "A: Mocha, Chai and a cup of Expresso"
+  },
+  {
+    "pun": "Q: Which planet in the solar system would a python developer first travel to?",
+    "punchline": "A: Jupyter"
+  },
+  {
+    "pun": "Q: Why did the developer turn down the marriage proposal?",
+    "punchline": "A: It’s a git commitment"
```

Comment:
Loved all of them except this one. Could you remove this one please? Or come up with a better punchline for it?

Reply:
No problem! I'm changed the git pun. 
I'm not doing committing (via VSCode on my laptop) and doing this directly on the browser on another computer. If I'm not doing anything correctly please let me know :)